resources/scripts/worker.js

```diff
@@ -1,5 +1,3 @@
-importScripts('fuse.min.js');
```

Comment:
For usual JS testing, you can check out the `Console` on your browser (Inspect > Console option appears). However, testing for worker js files would have to be done on server - you can push to your fork, publish that branch in your fork as a `gh-pages` website. Then, have a look at ayuhsya.github.io/mfqp/ and check the Console of that website.